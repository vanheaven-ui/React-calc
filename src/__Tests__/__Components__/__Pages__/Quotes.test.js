import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Quotes from '../../../components/pages/Quote';

test('is rendered without crashing', () => {
  const rootNode = document.createElement('div');
  ReactDOM.render(
    <BrowserRouter><Quotes /></BrowserRouter>, rootNode
  );
});

test('doesnot change unexpectedly', () => {
  const quotesTree = renderer.create(
    <BrowserRouter><Quotes /></BrowserRouter>
  ).toJSON();
  expect(quotesTree).toMatchSnapshot();
});
