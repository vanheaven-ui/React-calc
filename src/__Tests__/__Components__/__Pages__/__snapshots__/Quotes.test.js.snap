// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`doesnot change unexpectedly 1`] = `
Array [
  <nav
    className="navbar"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      <h2>
        Math Magicians
      </h2>
    </a>
    <div
      className="links"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
      <a
        href="/calc"
        onClick={[Function]}
      >
        Calculator
      </a>
      <a
        href="/quotes"
        onClick={[Function]}
      >
        Quotes
      </a>
    </div>
  </nav>,
  <article
    className="quotes-page main"
  >
    <h1>
      Enjoy this famous quote!
    </h1>
    <figure>
      <blockquote>
        Without Mathematics, there is nothing you can do. Everything around you is mathematics.
        <br />
        Everything around you is numbers
        <br />
        <br />
        <span>
          _By: Shakuntala Devi
        </span>
      </blockquote>
    </figure>
  </article>,
]
`;
